### ISSUE TITLE
Pfam labels

### ISSUE THREAD

**Issue by timkucera** [BorgwardtLab/proteinshake#254]
For pfam, there are some labels in the validation/test that are not present in the training set

**Comment by mahdip72** [comment]
This is my problem as well.
@timkucera I have a question, the paper says that family classification is a **multi class task** but some proteins have multiple Pfam annotations.  For example. I have the following labels for one protein:
```
['PF02324', 'PF19127']
```

Could you explain to me which annotation should I consider? Or is it a **multi label** task actually.
